cpp/log/cert.cc

```diff
@@ -1120,30 +1122,33 @@ Cert::Status TbsCertificate::DeleteExtension(int extension_nid) {
 
   // ExtensionIndex returns the first matching index - if the extension
   // occurs more than once, just give up.
-  int ignored;
-  status = ExtensionIndex(extension_nid, &ignored);
-  if (status == Cert::TRUE) {
+  status = ExtensionIndex(extension_nid);
+  if (status.ok() || status.status().CanonicalCode() == Code::INTERNAL) {
```

Comment:
We shouldn't get INTERNAL here because it's the same NID we looked up once before without seeing this error and that return implies it wasn't known. However, now that I read it again that INTERNAL status will be propagated by the next if condition. I think it's more readable to remove the INTERNAL check and not worry about the log message. I'll do that.